app/Http/Controllers/Api/AccessoriesController.php

```diff
@@ -27,9 +27,23 @@ class AccessoriesController extends Controller
     public function index(Request $request)
     {
         $this->authorize('view', Accessory::class);
-        $allowed_columns = ['id','name','model_number','eol','notes','created_at','min_amt','company_id'];
-
-        $accessories = Accessory::with('category', 'company', 'manufacturer', 'users', 'location');
+        
+        // This array is what determines which fields should be allowed to be sorted on ON the table itself, no relations
+        // Relations will be handled in query scopes a little further down.
+        $allowed_columns = 
+            [
+                'id',
+                'name',
+                'model_number',
+                'eol',
+                'notes',
+                'created_at',
+                'min_amt',
+                'company_id'
+            ];
```

Comment:
Love these comments *and* this column-name per-line thing, so much easier to work with!